draft-ietf-quic-multipath.md

```diff
@@ -575,6 +575,14 @@ any of which it can use when sending packets on that path; see also {{Section 5.
 Usually, it is desired to provide at least one additional connection ID for
 all used paths, to allow for migration.
 
+Over a given path, both endpoints use Connection IDs associated to a given Path
+ID. To initiate a path, each endpoint needs to advertise at least one Connection ID
+for a given Path ID to its peer. Endpoints SHOULD NOT create Path ID discontinuity
+through their Connection ID advertisements. For instance, if the Maximum Path ID
+limit is 2 and the endpoint wants to provide Connection IDs for only one Path ID,
+it should select Path ID 1 (and not Path ID 2). Similarly, endpoints SHOULD consume
+Path IDs in a continuous way, i.e., when creating paths.
```

Comment:
Not quite; this is not because a Path ID is valid (e.g., if max path id is 2, Path IDS 0 1 and 2 are valid) that you can use it nor "generate" it. A Path ID becomes usable once Connection IDs associated to that specific Path ID are advertised. We should ensure that the way we propose Path IDs through MP_NEW_CONNECTION_IDs do not create such discontinuity (e.g., client only provide CIDs for Path ID 0 and 2, while server only provide CIDs for Path ID 0 and 1); but I haven't found a better way to phrase this yet.

Reply:
I agree with your point. 
Some minor fix. About the example: We jumped out of Path ID 0 because it's the initial Path ID, but it's still an available choice for endpoints.